src/toil/utils/toilLaunchCluster.py

```
@@ -84,14 +84,16 @@ def main():
         try:
             from toil.provisioners.aws.awsProvisioner import AWSProvisioner
         except ImportError:
-            raise RuntimeError('The aws extra must be installed to use this provisioner')
+            print('The aws extra must be installed to use this provisioner')
+            raise
```

I don't disapprove, Just wondering why this change.

---

It is clearer that way. Hiding exceptions by raising a different error is not a good practice.

---

I agree.  A plain `raise` is almost always better.

---

Is this a program or a library?  print from libraries is generally not a good idea, they should go through the logger instead.

If this is a program, error messages should always be printed to stderr, not stdout.

I don't completely have the context here.  How are tests reporting error>

---

Program. Stderr is better.